Code/Tools/AssetProcessor/native/utilities/PlatformConfiguration.cpp

```diff
@@ -1125,6 +1141,17 @@ namespace AssetProcessor
 
         MetaDataTypesVisitor visitor;
         settingsRegistry->Visit(visitor, AZ::SettingsRegistryInterface::FixedValueString(AssetProcessorSettingsKey) + "/MetaDataTypes");
+
+        using namespace AzToolsFramework::AssetUtils;
+        AZStd::vector<AZStd::string> supportedFileExtensions;
+        AssetImporterPathsVisitor assetImporterVisitor{ settingsRegistry, supportedFileExtensions };
+        settingsRegistry->Visit(assetImporterVisitor, AZ::SettingsRegistryInterface::FixedValueString(AssetImporterSettingsKey) + "/" + AssetImporterSupportedFileTypeKey);
+
+        for (auto& entry : assetImporterVisitor.m_supportedFileExtensions)
+        {
+            visitor.m_metaDataTypes.push_back({ AZStd::string::format("%s.assetinfo", entry.c_str()), entry });
```

Comment:
So is this adding *.stl.assetinfo as a metadata type? Why can't we just say *.assetinfo is a metadata type?

Reply:
Oh I see, this is pairing the specific .assetinfo to .stl.